docs/src/main/paradox/index.md

```diff
@@ -321,6 +324,16 @@ Alpakka Kafka lets you connect Apache Kafka to Akka Streams.
     version=$alpakka-kafka.version$
 }
 
+## Akka Edge Rust $akka-edge-rust.version$ *@ref[Incubating](support-terminology.md#incubating)*
+
+*Extending the power of Akka’s event-driven model to resource-constrained devices.*
+
+Akka Edge support in Rust is designed to empower Akka developers at the edge. Akka Edge Rust brings the concepts of Command and Query Responsibility Segregation (CQRS) to developers familiar with Akka Persistence and Projections recognizing the resource constraints present at the edge.
```

Comment:
I'm not sure CQRS is the key thing we want to promote for this. Maybe:

Akka Edge Rust brings the concepts event-driven architecture to developers familiar with Akka Persistence and Projections recognizing the resource constraints present at the edge.